src/app/content/practiceQuestions/components/Answer/index.tsx

```diff
@@ -68,19 +69,31 @@ const Answer = ({
   // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [showCorrect, isCorrect]);
 
+  const onKeyDown = (e: React.KeyboardEvent<HTMLElement>) => {
+    // Space
+    if (e.which === 32) {
```

Comment:
you really should not be having to manually process these key events

Reply:
it doesn't work without this

Reply:
ok i'll look into it